### PR TITLE
Rename "model" fields with "dtdlModel" or "dtmiModel" to be more descriptive

### PR DESCRIPTION
**Pull request by bikamani**

### CODE REVIEW

File: sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java [hunk 1]
```diff
@@ -49,24 +49,24 @@
     private boolean decommissioned;
 
     /*
-     * The model definition.
+     * The model definition that conforms to the Digital Twins Definition Language (DTDL) v2.
```
Comment (drwill-ms) [inline sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java:52]: remove the word "the" before "Digital Twins Definition Language"

**Review by timtay-microsoft — APPROVED** [review]

File: sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java (hunk 1 above)
Comment (bikamani) [reply sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java:52]: done

Comment (drwill-ms) [inline sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java]: Can you add a link to the documentation, like I did in the .NET PR? Check your work item for a link.

Comment (bikamani) [reply sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/DigitalTwinsModelData.java]: I will take care of this in next PR.

**Pull request merged**